### PR TITLE
Removes 049 NPC behavior stuff

### PR DESCRIPTION
**Pull request by Vivalas**
## About the Pull Request
Removes SCP-049's NPC behavior.

## Why It's Good For The Game
It makes you randomly walk around even if you have a client. To be honest, I could fix it but I don't see much purpose. SCP-049 has lore describing him as completely still for a long time if he wants to be, and most players will always pick him up right away.

## Changelog

:cl:

code: Removed SCP049's automated movement code.

/:cl:

### CODE REVIEW

**Comment by Vivalas** [comment]
guh what in the fuck

**Comment by Tsurupeta** [comment]
imagine not fucking up your history

**Comment by Vivalas** [comment]
I honestly am just pressing buttons hoping one of them fixes things

**Comment by Vivalas** [comment]
today I learned how github works!

File: code/modules/SCP/SCPs/SCP-049.dm [hunk 1]
```diff
@@ -112,85 +112,10 @@ GLOBAL_LIST_EMPTY(scp049_1s)
...
-							target = L
-							return target
-					if (3)
-						// pick any target
-						target = L
-						return target
-	return target
+	return
 
 /mob/living/carbon/human/scp049/proc/pursueTarget()
```
Comment (ivanmixo) [inline code/modules/SCP/SCPs/SCP-049.dm]: Why keep these?

File: code/modules/SCP/SCPs/SCP-049.dm [hunk 2]
```diff
@@ -112,85 +112,10 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 	playsound(src, pick(voiceline), 30)
 
 /mob/living/carbon/human/scp049/proc/getTarget()
```
Comment (ivanmixo) [inline code/modules/SCP/SCPs/SCP-049.dm]: Why?

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/outsider/scp049.dm [hunk 3]
```diff
@@ -37,25 +37,6 @@
 /datum/species/scp049/handle_post_spawn(mob/living/carbon/human/H)
 	. = ..()
```
Comment (ivanmixo) [inline code/modules/mob/living/carbon/human/species/outsider/scp049.dm]:
```suggestion
```
This also has no reason to exist.

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**